kernel/kernel_config

```diff
@@ -2892,14 +2930,20 @@ CONFIG_EXPORTFS=y
 # CONFIG_EXPORTFS_BLOCK_OPS is not set
 CONFIG_FILE_LOCKING=y
 CONFIG_MANDATORY_FILE_LOCKING=y
-# CONFIG_FS_ENCRYPTION is not set
+CONFIG_FS_ENCRYPTION=y
```

Comment:
to confirm my understanding: this option enables the possibility for fs encryption, but doesn't actually perform it right?  The docs page is a bit terse but seems to read that way